packages/broker/src/helpers/PayloadFormat.ts

```diff
@@ -0,0 +1,94 @@
+export interface PayloadFormat {
+    createMessage: (payload: string) => Message
+    createPayload: (content: any, metadata?: Metadata) => string
```

Comment:
May be worth adding `never` to document that error could be thrown.

```suggestion
    createMessage: (payload: string) => Message | never
    createPayload: (content: any, metadata?: Metadata) => string | never
```

Reply:
Fixed